channeldb/payment_control.go

```diff
@@ -148,13 +152,14 @@ func (p *PaymentControl) InitPayment(paymentHash lntypes.Hash,
 		// retrying the payment or return a specific error.
 		case err == nil:
 			switch paymentStatus {
-
 			// We allow retrying failed payments.
 			case StatusFailed:
 
-			// This is a new payment that is being initialized for
-			// the first time.
-			case StatusUnknown:
+			// We already have payment creation info for this
+			// payment so we won't allow creating a duplicate one.
+			case StatusInitiated:
```

Comment:
Perhaps swap this commit with the next so that the sub-project of removing `StatusUnknown` is wrapped up first.

Reply:
Switched and merged with the previous commit!